File: app/src/main/java/com/wangzhen/reader/utils/Charset.java
package com.wangzhen.reader.utils;

/**
 * charsets
 * Created by wangzhen on 2023/4/12
 */
public enum Charset {
    UTF8("UTF-8"), GBK("GBK");

    private final String mName;
    public static final byte BLANK = 0x0a;

    Charset(String name) {
        mName = name;
    }

    public String getName() {
        return mName;
    }
}

File: app/src/main/java/com/wangzhen/reader/utils/Charset.kt
package com.wangzhen.reader.utils

/**
 * charsets
 * Created by wangzhen on 2023/4/12
 */
enum class Charset(val charset: String) {
    UTF8("UTF-8"), GBK("GBK");

    companion object {
        const val BLANK: Byte = 0x0a
    }
}